src/data/screenshots.json

```diff
@@ -219,201 +219,201 @@
                 ]
             },
             {
                 "title": "Test Certificate",
                 "anchor": "ios_test_certificate",
                 "images": [
-                    { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-homescreenrisk_level_low_installation_14days_0001", "alt": "Test certificate image 1 of 5" },
-                    { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-submissionflow_screenshot_test_certificate_entered_birthday-EN", "alt": "Test certificate image 2 of 5" },
-                    { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-screenshot_test_certificate_overview_part1", "alt": "Test certificate image 3 of 5" },
-                    { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-screenshot_test_certificate_valid_details_part1", "alt": "Test certificate image 4 of 5" },
-                    { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-screenshot_test_certificate_valid_details_part2", "alt": "Test certificate image 5 of 5" }
+                    { "filename": "/assets/screenshots/2-11/en/ios/iPhone 11 Pro-homescreenrisk_level_low_installation_14days_0001", "alt": "Test certificate image 1 of 5" },
+                    { "filename": "/assets/screenshots/2-11/en/ios/iPhone 11 Pro-submissionflow_screenshot_test_certificate_entered_birthday-EN", "alt": "Test certificate image 2 of 5" },
+                    { "filename": "/assets/screenshots/2-11/en/ios/iPhone 11 Pro-screenshot_test_certificate_overview_part1", "alt": "Test certificate image 3 of 5" },
+                    { "filename": "/assets/screenshots/2-11/en/ios/iPhone 11 Pro-screenshot_test_certificate_valid_details_part1", "alt": "Test certificate image 4 of 5" },
+                    { "filename": "/assets/screenshots/2-11/en/ios/iPhone 11 Pro-screenshot_test_certificate_valid_details_part2", "alt": "Test certificate image 5 of 5" }
                 ]
             },
             {
                 "title": "Vaccination Certificate",
                 "anchor": "ios_vaccination_certificate",
                 "images": [
-                    { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-screenshot_vaccination_certificate_overview_part1", "alt": "Vaccination certificate image 1 of 3" },
-                    { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-screenshot_second_vaccination_certificate_details_part1", "alt": "Vaccination certificate image 2 of 3" },
-                    { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-screenshot_second_vaccination_certificate_details_part2", "alt": "Vaccination certificate image 3 of 3" }
+                    { "filename": "/assets/screenshots/2-11/en/ios/iPhone 11 Pro-screenshot_vaccination_certificate_overview_part1", "alt": "Vaccination certificate image 1 of 3" },
```

Comment:
This screenshot was not taken on version 2.11, it should be replaced with a screenshot taken on version 2.11.